### ISSUE TITLE
Documentation out of date?

### ISSUE THREAD

**Issue by HardDiskDog** [suloku/gcmm#54]
the "apps" folder described to be in the zip isn't present in the latest version, but there is an "src" folder that the instructions don't mention, both on github and the readme file

**Comment by suloku** [comment]
You are correct. The src folder is the source code for the released version.
Inside the wii folder there is a gcmm and gcmm-dark folders, those two folders are the ones that go into the apps folder in your wii's sdcard (just like any other wii homebrew).
It seems I forgot to nest them into an apps folder, I'll update the zip file later.

**Comment by ursi** [comment]
Just noticed this myself. It is still outdated on the current release.
